src/editor/parts.js

```diff
@@ -426,7 +426,7 @@ export class PartCreator {
         let room;
         if (alias[0] === '#') {
             room = this._client.getRooms().find((r) => {
```

Comment:
`getRooms` here could be `getVisibleRooms` but the former is cheaper and shouldn't cause issues